ParseSwift.playground/Pages/2 - Finding Objects.xcplaygroundpage/Contents.swift

```diff
@@ -54,7 +54,11 @@ query.limit(2).find(callbackQueue: .main) { results in
         }
 
     case .failure(let error):
-        assertionFailure("Error querying: \(error)")
+        if error.equalsTo(.objectNotFound) {
+            assertionFailure("Object not found for this query")
```

Comment:
Did you build and test all of these in playgrounds? Did you make errors occur to see what happens?

Reply:
Can you answer this question?